### PR TITLE
chore(data): new package com.xrtk.webxr package

### PR DESCRIPTION
**Pull request by StephenHodgson**
- Updated all xrtk packages
- Added com.xrtk.webxr package

### CODE REVIEW

**Comment by StephenHodgson** [comment]
@favoyang everything good here?

I was trying to trigger the bot to come and merge it but I guess I messed that up somehow.

File: data/packages/com.xrtk.gltf.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ topics:
   - modeling
   - utilities
 hunter: StephenHodgson
-readme: 'main:README.md'
-displayName_zhCN: 'XRTK.glTF'
-description_zhCN: ''
```
Comment (favoyang) [inline data/packages/com.xrtk.gltf.yml:18]: Please keep the displayName_zhCN and description_zhCN field.

File: data/packages/com.xrtk.lenovo.yml [hunk 2]
```diff
@@ -9,5 +9,6 @@ topics:
   - ar-and-vr
 hunter: StephenHodgson
 createdAt: 1589834675554
-displayName_zhCN: 联想
-description_zhCN: XRTK的Lenovo平台组件
```
Comment (favoyang) [inline data/packages/com.xrtk.lenovo.yml:13]: Please keep the displayName_zhCN and description_zhCN field.

File: data/packages/com.xrtk.lumin.yml [hunk 3]
```diff
@@ -2,12 +2,13 @@ name: com.xrtk.lumin
 displayName: XRTK.Lumin
 description: The Lumin platform components for the XRTK
 repoUrl: 'https://github.com/XRTK/Lumin'
+parentRepoUrl: 'https://github.com/XRTK/XRTK-Core'
```
Comment (favoyang) [inline data/packages/com.xrtk.lumin.yml:5]: The parentRepoUrl is wrong. The repository is not a fork.

File: data/packages/com.xrtk.webxr.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+name: com.xrtk.webxr
+displayName: XRTK.WebXR
+description: The WebXR platform components for the XRTK
+repoUrl: 'https://github.com/XRTK/WebXR'
+parentRepoUrl: 'https://github.com/XRTK/XRTK-Core'
```
Comment (favoyang) [inline data/packages/com.xrtk.webxr.yml:5]: The parentRepoUrl is wrong. The repository is not a fork.

File: data/packages/com.xrtk.wmr.yml [hunk 5]
```diff
@@ -2,12 +2,13 @@ name: com.xrtk.wmr
 displayName: XRTK.WindowsMixedReality
 description: The Windows Mixed Reality platform components for the XRTK
 repoUrl: 'https://github.com/XRTK/WindowsMixedReality'
+parentRepoUrl: 'https://github.com/XRTK/XRTK-Core'
```
Comment (favoyang) [inline data/packages/com.xrtk.wmr.yml:5]: The parentRepoUrl is wrong. The repository is not a fork.

File: data/packages/com.xrtk.lumin.yml [hunk 6]
```diff
@@ -2,12 +2,13 @@ name: com.xrtk.lumin
...
 licenseSpdxId: MIT
 licenseName: MIT License
 topics:
   - ar-and-vr
 hunter: favoyang
-readme: 'main:README.md'
 createdAt: 1577885136000
-displayName_zhCN: XRTK.Lumin
-description_zhCN: XRTK的Lumin平台组件
```
Comment (favoyang) [inline data/packages/com.xrtk.lumin.yml:13]: Please keep the displayName_zhCN and description_zhCN field.

File: data/packages/com.xrtk.nreal.yml [hunk 7]
```diff
@@ -9,5 +9,7 @@ topics:
   - ar-and-vr
 hunter: StephenHodgson
 createdAt: 1589834675554
-displayName_zhCN: XRTK.Nreal
-description_zhCN: XRTK的Nreal平台组件
```
Comment (favoyang) [inline data/packages/com.xrtk.nreal.yml:13]: Please keep the description_zhCN field.

File: data/packages/com.xrtk.wmr.yml [hunk 8]
```diff
@@ -2,12 +2,13 @@ name: com.xrtk.wmr
...
 licenseSpdxId: MIT
 licenseName: MIT License
 topics:
   - ar-and-vr
 hunter: favoyang
-readme: 'main:README.md'
 createdAt: 1577885177000
-displayName_zhCN: XRTK.WindowsMixedReality
-description_zhCN: XRTK的Windows Mixed Reality平台组件
```
Comment (favoyang) [inline data/packages/com.xrtk.wmr.yml:13]: Please keep the displayName_zhCN and description_zhCN field.

**Review by favoyang — CHANGES_REQUESTED** [review]
Please fix the review first.

**Comment by StephenHodgson** [comment]
@favoyang all fixed :)

**Review by favoyang — APPROVED** [review]
LGTM

**Pull request merged**